src/main.rs

```diff
@@ -71,31 +302,161 @@ fn main() -> OrtResult<()> {
         })
         .into_dyn(),
     );
-
-    // Prepare an input batch for inference with a single instance
     let inputs = vec![Value::from_array(session.allocator(), &input_tensor)?];
-
-    // Perform the inference
     let outputs = session.run(inputs)?;
-
-    // Extract the output tensor from the output batch
     let output_tensor = outputs[0].try_extract::<f32>()?;
-
-    // Update the alpha channel of the resized image with the predicted values
     for (indices, alpha) in output_tensor.view().indexed_iter() {
         resized_img[(indices[3] as u32, indices[2] as u32)][3] = (alpha * 255.) as u8;
     }
-
-    // Resize the final output image back to the original size if possible using the scaling factor
     let output_img = imageops::resize(
         &resized_img,
         (input_img.width() as f32 * scaling_factor) as u32,
         (input_img.height() as f32 * scaling_factor) as u32,
         imageops::FilterType::Triangle,
     );
+    Ok(DynamicImage::ImageRgba8(output_img))
+}
 
-    // Save the final output image to the output image file path
-    output_img.save(output_img_file).unwrap();
+async fn start_http_server() -> Result<(), anyhow::Error> {
+    let session = onnx_session().unwrap();
+    SESSION.set(session).ok();
+    // Define a closure to handle incoming HTTP requests
+    let make_svc = make_service_fn(|_conn| async {
+        Ok::<_, Infallible>(service_fn(move |req| handle_post(req)))
+    });
 
+    let addr = ([0, 0, 0, 0], 9876).into();
```

Comment:
should allow users to configure the address and port instead of fixed numbers

Reply:
addressed in [2059895](https://github.com/dnanhkhoa/rust-background-removal/pull/3/commits/20598956a8b3d1e8f359774066c2d4526a364a2e)